docs/gwcs/points_to_wcs.rst

```diff
@@ -0,0 +1,51 @@
+.. _user_api:
```

Comment:
This label already exists. Replace it with something else. This is causing one of the failures on Travis.